iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/functionclass/EventDataListener.java

```diff
@@ -66,6 +71,8 @@ public EventDataListener(final Device device, final Scheduler scheduler, final D
                     scheduleNr = this.extractScheduleIndex(bda.getReference());
                 }
 
+                RelaySchedulePair relaySchedulePairs = new RelaySchedulePair(relayNr, scheduleNr);
```

Comment:
relaySchedulePairs => relaySchedulePair, because it contains exactly one pair.

Reply:
Done